<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Questions</class>
 <widget class="QDialog" name="Questions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="homeButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QPushButton" name="visualiseButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>0</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Visualise</string>
   </property>
  </widget>
  <widget class="QPushButton" name="questionsButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>0</y>
     <width>89</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Questions</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>371</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Select one of the following question:</string>
   </property>
   <widget class="QRadioButton" name="addOption">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>471</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Why A is not involved in the plan?</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="removeOption">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>481</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Why A is involved in the plan?</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="replaceOption">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>491</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Why A insted of B? </string>
    </property>
   </widget>
   <widget class="QRadioButton" name="windowOption">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>461</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Why A not at time t[i]?</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="actionOption">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>501</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Why A not before B?</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="actionOption2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Why A not after B?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="choiceButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>190</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="question_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="planArea">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>360</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="question_label1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>341</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="list">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>70</y>
     <width>161</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="doneButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>290</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
